Proton/Sources/Core/RichTextEditorContext.swift

```diff
@@ -67,21 +67,20 @@ class RichTextEditorContext: RichTextViewContext {
                 return false
             }
 
-            guard range.endLocation <= textView.attributedText.length else { return false }
-
-            let substring = textView.attributedText.attributedSubstring(from: range)
```

Comment:
@rjchatfield, can you please confirm the scenario where this code crashes?

Reply:
No I can not. The screenshot in the description is from production - it was seen to crash on that line many times, but we weren't able to reproduce it locally. As mentioned, I can't see why this crashes.